src/bots/kleros-liquid.js

```diff
@@ -154,7 +153,7 @@ module.exports = async (web3, batchedSend) => {
       readyForNextPhase = true
     } else if (phase == PhaseEnum.drawing) {
       const maxDrawingTime = await klerosLiquid.methods.maxDrawingTime().call()
-      if ((Date.now() - lastPhaseChange * 1000 >= maxDrawingTime * 1000) && disputesWithoutJurors == 0) {
+      if ((Date.now() - lastPhaseChange * 1000 >= maxDrawingTime * 1000) || disputesWithoutJurors == 0) {
```

Comment:
To match the `KlerosLiquid.passPhase()` logic here: https://github.com/kleros/kleros/blob/master/contracts/kleros/KlerosLiquid.sol#L402